RELEASING.md

```diff
@@ -1,12 +1,11 @@
 Releasing
 =========
 
- 1. Change the version in `gradle.properties` to a non-SNAPSHOT version.
+ 1. Change the version in `gradle.properties` to your desired release version
  2. Update the `CHANGELOG.md` for the impending release.
- 3. `git commit -am "Prepare for release X.Y.Z."` (where X.Y.Z is the new version)
+ 3. `git commit -am "Create release X.Y.Z."` (where X.Y.Z is the new version)
  4. `git tag -a X.Y.Z -m "Version X.Y.Z"` (where X.Y.Z is the new version)
- 5. `./gradlew clean uploadArchives`
- 6. Update the `gradle.properties` to the next SNAPSHOT version.
- 7. `git commit -am "Prepare next development version."`
- 8. `git push && git push --tags`
- 9. Visit [Sonatype Nexus](https://oss.sonatype.org/) and promote the artifact.
+ 5. Change the version in `gradle.properties` to the next SNAPSHOT version
+ 6. `git commit -am "Prepare next snapshot version"`
```

Comment:
Should we maybe say "Prepare version X.Y.Z snapshot"?

Reply:
The reason i say next here is because the convention is that the SNAPSHOT version is usually higher than the last stable release. 
It shouldnt be X.Y.Z-SNAPSHOT, but rather X.Y.(Z+1)-SNAPSHOT